inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/manager/enums/WorkflowState.java

```diff
@@ -15,19 +15,31 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package de.tudarmstadt.ukp.inception.workload.dynamic.manager.enums;
 
-package de.tudarmstadt.ukp.inception.workload.dynamic.manager;
+import de.tudarmstadt.ukp.clarin.webanno.support.PersistentEnum;
 
-/**
- * @deprecated This needs to be replaced by a proper extension point for workload management
- *             strategies as soon as possible.
- */
-@Deprecated
-public interface WorkloadProperties
+public enum WorkflowState implements PersistentEnum
 {
-    boolean isWorkloadManagerActive();
+    DYNAMIC_WORKFLOW("DYNAMIC_WORKFLOW"),
```

Comment:
Is it possible to move enum somewhere else, e.g. the central/default workflow place? The dynamic workflow module should not define the default one. Also, I think @reckart  does not like enums but would prefer string constants.

Reply:
We use DB-manageable enums for states. Cf. `SourceDocumentState` and `SourceDocumentStateType`.

Reply:
But... this does not seem to be a state. It seems to be a plugin ID. That needs do be done differently because we cannot have a single place where all the plugins are hard-coded. That would go against the idea of having plugins.